pom.xml

```diff
@@ -2511,8 +2511,11 @@
 
     <profile>
       <id>hadoop-2.7</id>
+      <activation>
```

Comment:
How can I set this profile for an SBT build/test run? I just want to have SBT doing it as otherwise the patch is unverified by the machinery. And nobody wants that —do they?